contracts/src/CrossChainApplications/NativeTokenBridge/NativeTokenDestination.sol

```diff
@@ -0,0 +1,192 @@
+// (c) 2023, Ava Labs, Inc. All rights reserved.
```

Comment:
Lets make a unit test suite for this contract as well as `NativeTokenSource.sol`

Reply:
Bumping this comment :) 

I'd consider solidity unit test suites for these contracts a requirement for merging this PR so that we can be confident that the core logic is correct. Here's an example for the ERC20Bridge contracts.

https://github.com/ava-labs/teleporter/blob/main/contracts/src/CrossChainApplications/ERC20Bridge/tests/ERC20BridgeTests.t.sol